src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.Properties.cs

```diff
@@ -135,6 +141,12 @@ public string Text
 			typeof(global::Windows.UI.Xaml.Controls.AutoSuggestBox),
 			new FrameworkPropertyMetadata(default(global::Windows.UI.Xaml.Controls.IconElement), propertyChangedCallback: (s, e) => (s as AutoSuggestBox)?.UpdateQueryButton()));
 
+		public static global::Windows.UI.Xaml.DependencyProperty DescriptionProperty { get; } =
+			Windows.UI.Xaml.DependencyProperty.Register(
+				nameof(Description), typeof(object),
+				typeof(global::Windows.UI.Xaml.Controls.AutoSuggestBox),
+				new FrameworkPropertyMetadata(default(object), propertyChangedCallback: (s, e) => (s as AutoSuggestBox)?.UpdateDescriptionVisibility(false)));
```

Comment:
Have you confirmed whether the default value is an empty string or null?

Reply:
It is `null` as it is an `object`, by default the presenter is invisible